src/main/kotlin/no/nav/klage/oppgave/clients/MicrosoftGraphClient.kt

```diff
@@ -36,6 +38,7 @@ class MicrosoftGraphClient(private val microsoftGraphWebClient: WebClient) {
             .block()?.onPremisesSamAccountName ?: throw RuntimeException("NavIdent could not be fetched")
     }
 
+    @Retryable
```

Comment:
Burde denne vært på `getDisplayNames()` i stedet?

Reply:
Jeg er i og for seg enig i det, men da må vi endre koden en god del. Det er ikke bare at det ikke fungerer på private-metoder men det fungerer ikke på kall internt i en klasse heller. Så vi må splitte det opp i to, dra noe av det ut i en annen klasse. Skal jeg gjøre det?

Reply:
Har innført et SaksbehandlerRepository for å få et nivå til, kunne da flytte annotasjonen